src/axom/slic/core/SimpleLogger.hpp

```diff
@@ -60,10 +61,10 @@ class SimpleLogger
   /*!
    * \brief Constructor initializes slic logging environment.
    */
-  SimpleLogger()
+  explicit SimpleLogger(message::Level level = message::Debug)
```

Comment:
This line contains the main change in this PR -- it adds an initial logging level for the `SimpleLogger`.